### PR TITLE
Refactor init process

### PR DESCRIPTION
**Pull request by nekia**
To remove our code changes around system chaincode, done some refactoring:

- Replaced the feature of block integrity check to seperated command (ledgerfsck)
- Fixed some warning pointed out by go-lint/go-vet

### CODE REVIEW

File: cmd/ledgerfsck/main.go [hunk 1]
```diff
@@ -283,12 +298,13 @@ func (fsck *ledgerFsck) GetLatestChannelConfigBundle() error {
 func (fsck *ledgerFsck) Verify() {
 	blockchainInfo, err := fsck.ledger.GetBlockchainInfo()
 	if err != nil {
-		logger.Errorf("could not obtain blockchain information "+
+		logger.Debugf("could not obtain blockchain information "+
```
Comment (anand-jpa) [inline cmd/ledgerfsck/main.go:301]: Why is this Error log entry changed to Debug?

**Review by anand-jpa — COMMENTED** [review]
Why are Error log entries changed to Debug and Info?

File: cmd/ledgerfsck/main.go (hunk 1 above)
Comment (nekia) [reply cmd/ledgerfsck/main.go:301]: That's for filtering log output from this command into 'FAIL' or 'PASS'. This command is mainly used from shell script and the script doesn't take care of the other additional messages. @anand-jpa

**Review by anand-jpa — APPROVED** [review]
Looks good to me

**Pull request merged**